### PR TITLE
Fixes #15: Added CodeQL Workflow

### LINKED ISSUE #15: Add CodeQL Analysis Workflow
Adding [CodeQL](https://github.com/github/codeql-action) workflow in repo.

If this seems fine please assign this to me

### PR DESCRIPTION
**Pull request by uzaxirr**
It's similar to one in [IntelOwl](https://github.com/intelowlproject/IntelOwl/blob/master/.github/workflows/codeql-analysis.yml)

Fixes and Closes #15 

## Screenshots
Screenshot from 2022-01-02 16-01-16

### CODE REVIEW

File: .github/workflows/Create codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      with:
+        # We must fetch at least the immediate parents so that if this is
+        # a pull request then we can checkout the head.
+        fetch-depth: 2
+
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
```
Comment (mlodic) [inline .github/workflows/Create codeql-analysis.yml]: could you just set 3.9 as it is the version used by the application?

File: .github/workflows/Create codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends apt-utils libsasl2-dev libssl-dev libldap2-dev python-dev libfuzzy-dev net-tools git osslsigncode exiftool
```
Comment (mlodic) [inline .github/workflows/Create codeql-analysis.yml]: well this is just copy-pasted from IntelOwl. Greedybear does not have the same dependencies

File: .github/workflows/Create codeql-analysis.yml (hunk 1 above)
Comment (uzaxirr) [reply .github/workflows/Create codeql-analysis.yml]: Done!

File: .github/workflows/Create codeql-analysis.yml (hunk 2 above)
Comment (uzaxirr) [reply .github/workflows/Create codeql-analysis.yml]: Yeah, I was about to ask that. I thought it's based on the same tech stack so I assumed it to consume those dependencies. Thanks for the clarification.

**Pull request merged**